spec/services/partner_fetch_requestable_items_service_spec.rb

```diff
@@ -21,20 +37,22 @@
       end
 
       it 'should return all active and visible items' do
-        expected_items = organization.items.active.visible.map { |item| [item.name, item.id] }.sort
+        expected_items = [[items[0].name, items[0].id]]
```

Comment:
We haven't really gained much here - we are still relying on the auto-generated name instead of specifying one.

Reply:
This is a bit confusing to me. The test is essentially: give an org items that are the 4 combos of active and visible, then make sure the service only returns the ones we expect. Does it matter if the names are generate by factorybot? Or am I misunderstanding how to test this?

Reply:
The service returns those items by checking the database. If those items aren't in the database, this isn't a "real" test of the feature.

Reply:
Oh, I guess I misunderstood. These items are getting built. When create(:org, ...) runs it saves the list of items and all 4 get built. Something like `expect { organization }.to change { Item.count }.by(4)`  passes. I probably should rework how I approach that if it isn't clear. 

That being said. Assuming you accept that the items are getting built, is this manner of test fine?

Reply:
I think I got rabbit-holed :grin: My issue here is the testing of `name` - I prefer us supplying the name and comparing it against what we supplied, rather than relying on the use of Faker or factory-generated name. Does that make sense?